### DISCUSSION TITLE
2021.02.23 화 회의록

### DISCUSSION THREAD

**Discussion by hakju** [hakju/ios-cloud-notes#21] [General]
[ 계획 ]

1. TIL 꼭! 쓰기
2. STEP 1 코드 리뷰오면 같이 이야기해보기
3. STEP 2 진행하기에 앞서서 core data 클론 코딩, 공부하기
4. swift study class 공부하기

**Comment by soleJin** [comment]
+ step1 버튼추가
+ core data?
